docs/user/metacat/source/harvester.rst

```diff
@@ -1,7 +1,17 @@
+----
+
+.. deprecated:: 3.0.0
+    The Metacat Harvester is obsolete.
+
+    While some features remain visible, the old Metacat API has been disabled.
+    All features that depend on the old Metacat API are deprecated.
+
+----
+
```

Comment:
shouldn't we just remove the harvester docs altogether?

Reply:
If the harvester was removed, let's remove the docs. Presumably we no longer have the `harvester` subdirectories?

Reply:
This document has been deleted after discussing with Jing.